src/Jackett.Common/Indexers/Abstract/AvistazTracker.cs

```diff
@@ -112,12 +112,14 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                     release.MinimumRatio = 1;
                     release.MinimumSeedTime = 172800; // 48 hours
 
-                    var qLink = qRow.Find("a.torrent-filename"); ;
+                    var qLink = qRow.Find("a.torrent-filename");
+                    ;
                     release.Title = qLink.Text().Trim();
                     release.Comments = new Uri(qLink.Attr("href"));
                     release.Guid = release.Comments;
 
-                    var qDownload = qRow.Find("a.torrent-download-icon"); ;
+                    var qDownload = qRow.Find("a.torrent-download-icon");
+                    ;
```

Comment:
fix manually

Reply:
@ngosang you have marked your review as approved, but have these "fix manually" comments... so presumably you want changes? And I should hold back merging?

Reply:
Sorry for the confusion. This PR can be merged, but there are little details that I want to be fixed. @cadatoiva are you going to do the changes in this PR?

Reply:
No, they'll fix themselves in a future PR. I'm just going through each category of automated fixes right now. We'll keep note of the details and make sure they're fixed in the future.

Reply:
The scope of this PR is whitespace and encoding issues.

Reply:
@garfield69 you can merge